docs/src/formulation-details.md

```diff
@@ -0,0 +1,85 @@
+# Formulation Details
+
+This section provides references to understand the formulations as provided by PowerModels. The list is not meant as a literature discussion, but to give the main starting points to understand the implementation of the formulations.
+
+The history of AC OPF formulations up to 2012 is discussed in:
+- Cain, M. B., O’ Neill, R. P., & Castillo, A. (2012). History of optimal power flow and formulations. <https://www.ferc.gov/industries/electric/indus-act/market-planning/opf-papers/acopf-1-history-formulation-testing.pdf>
+
+A handbook for developing an understanding of the principles of convex relaxation applied to optimal power flow is:
+- Taylor, J. A. (2015). Convex optimization of power systems (1st ed.). Cambridge University Press.
```

Comment:
In place of this reference I would recommend,
- https://arxiv.org/abs/1807.07227
- https://ieeexplore.ieee.org/document/7271127/

Reply:
done!